python/py-roman/Portfile

```diff
@@ -1,41 +1,37 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
-PortGroup python 1.0
+PortSystem          1.0
+PortGroup           python 1.0
 
-set realname        roman
-name                py-${realname}
-version             2.0.0
-python.versions     26 27 33 34 35 36
+name                py-roman
+version             3.0
 categories-append   textproc
 license             PSF-2.1.1
 platforms           darwin
 supported_archs     noarch
 maintainers         {aronnax @lpsinger} openmaintainer
+
 description         Integer to Roman numerals converter
 long_description    ${description}
-homepage            https://pypi.python.org/pypi/${realname}/
-master_sites        pypi:[string index ${realname} 0]/${realname}/
-distname            ${realname}-${version}
-use_zip             yes
-
-checksums           md5     aa71d131eec16d45c030fd06a27c9d17 \
-                    sha1    7fd136f80cf780bbf802ffe1c4989a64ff469c75 \
-                    rmd160  a8ca66d5fafd340986a45332d861d6bae259fe1e
-
-if {${name} eq ${subport}} {
-    livecheck.type  regex
-    livecheck.regex [format "%s-%s" ${realname} {(\d+(?:\.\d+)*)}]
-} else {
-    depends_build   port:py${python.version}-setuptools
 
-    pre-activate {
-        if {[file exists ${python.pkgd}/roman.py]
-            && ![catch {set vers [lindex [registry_active py${python.version}-docutils] 0]}]
-            && [vercmp [lindex $vers 1] 0.6] < 0} {
-            registry_deactivate_composite py${python.version}-docutils "" [list ports_nodepcheck 1]
-        }
-    }
+homepage            https://pypi.python.org/pypi/${python.rootname}/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}/
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  39d72b580e07e5c49bdf3a7db9efe987f3ff8d60 \
+                    sha256  1cb1cfc9386ec39bd5dd3190c2c89c1b253140bccda16807aab3627cbb3f275d \
+                    size    5184
+
+python.versions     26 27 33 34 35 36 37
```

Comment:
It might be reasonable to remove the 2.6 and 3.3 support as long as you're updating this.

Reply:
same situation as for ```py-olefile```, it's a dependency for ```py-docutils```, which has the same maintainer. I am in favor of removing all traces of py26/py33 subports and variants involving python26/python33 irrespective of who maintains a port, but that's not my decision to make as a newbie here.

Reply:
Actually, py-roman no longer supports py26 and py33 since v3.0 (https://pypi.org/project/roman/#description).

Reply:
well, by looking at the actual commits it seems to me that statement only means that "we do not explicitly test py26/py33 anymore in our CI system and therefore don't guarantee that it will work". However, the actual test still pass so assuming these sufficiently cover the actual code it should be good for now.

Reply:
I think we should just be dropping these things. I understand that @jmroot doesn't want us to but he's the only person with that opinion. That said, we can wait until another round of commits for that argument.